extensions/llamacpp/processors/LlamaContextInitializer.cpp

```diff
@@ -0,0 +1,30 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "LlamaContextInitializer.h"
+#include "llama.h"
+
+namespace org::apache::nifi::minifi::extensions::llamacpp::processors {
+
+LlamaContextInitializer::LlamaContextInitializer() {
```

Comment:
there might be some collision between the "contexts", like we have a DefaultLllamaContext which contains a LlamaContextInitializer, we could call it LlamaBackendInitializer or similar

Reply:
Renamed in https://github.com/apache/nifi-minifi-cpp/pull/1903/commits/d3560bf6f64afac8b50ca32a39d6dd43ef77878a